aggregate/site/category.xml

```diff
@@ -277,10 +277,15 @@
     <category name="FuseTooling" />
   </feature>
 
-	<!-- JBIDE-24897 not ready yet <feature id="com.redhat.fabric8analytics.lsp.eclipse.feature">
+  <!-- JBIDE-24897 not ready yet <feature id="com.redhat.fabric8analytics.lsp.eclipse.feature">
     <category name="WebTools" />
   </feature> -->
 
+  <!-- JBIDE-26863 Quarkus Tooling -->
+  <feature id="org.jboss.tools.quarkus.feature">
+    <category name="CoreTools" />
```

Comment:
not sure about the correct category here.

Reply:
id is correct